ui/app/AppLayouts/Profile/views/EnsTermsAndConditionsView.qml

```diff
@@ -21,12 +21,23 @@ import AppLayouts.Profile.stores 1.0
 Item {
     id: root
 
+    required property var accountAssetsModel
     property EnsUsernamesStore ensUsernamesStore
     property string username: ""
 
     signal backBtnClicked()
     signal registerUsername()
 
+    QtObject {
+        id: d
+        readonly property var sntToken: ModelUtils.getByKey(root.accountAssetsModel, "tokensKey", root.ensUsernamesStore.getStatusTokenKey())
```

Comment:
This is dangerous. In case of a model reset the app could crash.
`ModelEntry` would help here. Same in `ui/app/AppLayouts/Profile/ProfileLayout.qml`